packages/List/src/list-item.scss

```diff
@@ -0,0 +1,274 @@
+  --ids-list-item-icon-member-size: #{tokens.$space-5};
+
+  /* Checkmark */
+  --ids-list-item-checkmark-color: #{tokens.$electric-blue-500};
+  --ids-list-item-checkmark-margin: 0 0 0 #{tokens.$space-2};
+
+  /* Premium */
+  --ids-list-item-premium-color: #{tokens.$dandelion-500};
+
+  /* Manager */
+  --ids-list-item-manager-color: #{tokens.$grey-500};
+  --ids-list-item-manager-margin: 0 0 0 #{tokens.$space-2};
+
+  /* Checkbox */
+  --ids-list-item-checkbox-size: #{tokens.$font-size-3};
+  --ids-list-item-checkbox-radius: #{tokens.$border-radius-sm};
+  --ids-list-item-checkbox-check: #{tokens.$samoyed};
+  --ids-list-item-checkbox-margin: 0.25rem #{tokens.$space-2} 0.25rem 0;
+  --ids-list-item-checkbox-background: #{tokens.$samoyed};
+  --ids-list-item-checkbox-background-disabled: #{tokens.$grey-200};
+  --ids-list-item-checkbox-background-checked: #{tokens.$electric-blue-500};
+  --ids-list-item-checkbox-background-checked-disabled: #{tokens.$grey-400};
+  --ids-list-item-checkbox-border: #{tokens.$grey-400};
+  --ids-list-item-checkbox-border-hover: #{tokens.$grey-800};
+  --ids-list-item-checkbox-border-checked: #{tokens.$electric-blue-500};
+  --ids-list-item-checkbox-check-width: 4px;
+  --ids-list-item-checkbox-check-height: 8px;
+
+  /* Description */
+  --ids-list-item-desc-color: #{tokens.$grey-600};
+  --ids-list-item-desc-font-size: #{tokens.$font-size-2};
+  --ids-list-item-desc-line-height: #{tokens.$line-height-xl};
+  --ids-list-item-desc-margin: #{tokens.$space-1} 0 0;
+
+  /* Member Name */
+  --ids-list-item-member-color: #{tokens.$grey-600};
+  --ids-list-item-member-font-weight: #{tokens.$font-weight-semi-bold};
+}
+
+@keyframes check-animation {
+  0% {
+    width: 0;
+    height: 0;
+    opacity: 0;
+  }
+
+  30% {
+    width: var(--ids-checkbox-check-width);
+    height: 0;
+    margin-top: 0;
+    opacity: 1;
+  }
+
+  100% {
+    width: var(--ids-checkbox-check-width);
+    height: var(--ids-checkbox-check-height);
+  }
+}
+
+%check {
+  width: var(--ids-checkbox-check-width);
+  height: var(--ids-checkbox-check-height);
+  border: 2px solid currentColor;
+  border-bottom: 0;
+  border-left: 0;
+  transform: scaleX(-0.9) rotate(135deg);
+  transform-origin: top left;
+}
+
+.ids-list-item {
+  align-items: center;
+  display: flex;
+  min-height: var(--ids-list-item-min-height);
+  padding: var(--ids-list-item-padding);
+  user-select: none;
+
+  &--selectable {
+    cursor: pointer;
+  }
+
+  &--compact {
+    min-height: var(--ids-list-item-min-height-compact);
+    padding: var(--ids-list-item-padding-compact);
+  }
+
+  &--member {
+    min-height: var(--ids-list-item-min-height-member);
+    padding: var(--ids-list-item-padding-member);
+  }
+
+  &--selected,
+  &--focused {
+    background: var(--ids-list-item-background-selected);
+    outline: 0;
+    box-shadow: none;
+  }
+
+  &--focused {
+    background: var(--ids-list-item-background-focused);
```

Comment:
The focus now has a border
 
<img width="501" alt="Capture d’écran, le 2023-02-09 à 13 43 03" src="https://user-images.githubusercontent.com/2379339/217907613-bff87abf-639b-429d-8a2e-37b1008d023f.png">

Reply:
I'm also confused with this though... In my code, the hover and keyboard focus have the same style. It doesn't make sense to be different. Plus it would be difficult to to separate the two. This is how it's done in all other select components.

Reply:
The border has been removed. the focus/hover style is the same style as selected. for now..